integration/sync/test_collision-handlers.js

```diff
@@ -0,0 +1,115 @@
+var MongoClient = require('mongodb').MongoClient;
+var assert = require('assert');
+var sinon = require('sinon');
+var _ = require('underscore');
+var defaultDataHandlersModule = require('../../lib/sync/default-dataHandlers');
+var dataHandlersModule = require('../../lib/sync/dataHandlers');
+
+var DATASETID = "collisionHandlersTest";
+var MONGODB_URL = "mongodb://127.0.0.1:27017/test";
+
+var mongodb;
+var dataHandlers;
+var defaultHandlers;
+
+function getCollisionData() {
+  return {
+    pre: { test: 'test1' },
+    post: { test: 'test2' },
+    hash: 'testhash',
+    ts: Date.now(),
+    uid: 'testUid'
+  };
+}
+
+module.exports = {
+  'before': function(done) {
```

Comment:
looks like this is the setup for your test only. In this case, better to have it inside another object. This way mocha will only run it before your tests. With the current setup, mocha will run this before ALL the tests